vocab_files/attribute_concepts/quadrat-number.ttl

```diff
@@ -5,13 +5,13 @@ PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
-<https://linked.data.gov.au/def/nrm/2514a05c-80a8-5100-9513-db50684ed499>
+<https://linked.data.gov.au/def/nrm/301b5716-6a42-4426-8675-6c46343500fe>
```

Comment:
Hi @arunherb , can you please add this attribute to the collection file?

Reply:
Done